src/gui/accountsettings.h

```diff
@@ -143,8 +143,8 @@ private slots:
     bool _wasDisabledBefore;
     AccountState *_accountState;
```

Comment:
@mgallien this one should also be set to `nullptr`?

Reply:
the reason is that the constructor is always putting a value in it so a default value here would be always overriden